.github/workflows/platform-install-postgres.yml

```diff
@@ -43,6 +43,6 @@ jobs:
           echo "cucumber.publish.enabled=true" > qaframework-bdd-tests/src/test/resources/cucumber.properties
           mvn clean install -DskipTests=true
           cp -f qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test-local.properties qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test.properties
-          qaframework-bdd-tests
```

Comment:
Is this fixing that issue

Reply:
yaap, the command is supposed to be `cd qaframework-bdd-tests`